src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Header() {
```

Comment:
Lets rename this  component to something more descriptive like Banner.

Reply:
Done!